.buildkite/pipeline.yml

```diff
@@ -1,7 +1,7 @@
 steps:
   - label: "Run tests (Nixpkgs)"
     command: |
-      echo "build:ci --host_platform=@rules_haskell//haskell/platforms:linux_x86_64_nixpkgs" > .bazelrc.local
+      echo "build:ci --host_platform=@io_tweag_rules_nixpkgs//nixpkgs/platforms:host" > .bazelrc.local
```

Comment:
This needs rebasing on `master` due to https://github.com/tweag/rules_haskell/pull/1301/files#diff-f55b2f358053ad76fb5ac3f776f78ef8R13-R14.